pulsar-broker/src/test/java/org/apache/pulsar/client/impl/MessageChunkingSharedTest.java

```diff
@@ -211,6 +212,32 @@ private static String createChunkedMessage(int numChunks) {
         return Schema.STRING.decode(payload);
     }
 
```

Comment:
Add a case when user provides sequence ids and how the messages are chunked and deduped.

Reply:
Done.